code/datums/components/crafting/misc.dm

```diff
@@ -23,3 +23,13 @@
 		/datum/reagent/toxin/coffeepowder = 10,
 	)
 	category = CAT_MISC
+
+/datum/crafting_recipe/corppaperslip
+	name = "Corporate Plastic Card"
+	result = /obj/item/paper/paperslip/corporate
+	time = 3 SECONDS
+	reqs = list(/obj/item/paper/paperslip = 1,
+				/obj/item/stack/sheet/plastic = 3
+	)
```

Comment:
```suggestion
	reqs = list(
		/obj/item/paper/paperslip = 1,
		/obj/item/stack/sheet/plastic = 3,
	)
```

do list multilining like this, everything else looks poor (note the trailing comma)